client/modules/food/reducers/packing.js

```diff
@@ -142,6 +159,21 @@ export default (state = {}, action) => {
         saving: false,
         saveError: null,
       }
+    case RECEIVED_REQUEST: {
```

Comment:
You can add this under `case UNPACK_REQUEST` above, then all request type actions will set the loading state. Same with the failure action below.